### DISCUSSION TITLE
Getting started with GitHub

### DISCUSSION THREAD

**Discussion by SilasChalwe** [community/community#134627] [New to GitHub] [answered]
### Select Topic Area

General

### GitHub Feature Area

Other

### Body

How can I get started with GitHub,am a New member

**Comment by elsycharles** [comment, marked as answer]
Hi @SilasChalwe 

1. First learn about github see that here [https://docs.github.com/en/get-started/start-your-journey/about-github-and-git](https://docs.github.com/en/get-started/start-your-journey/about-github-and-git) 
2. Now you have an account, you can first personalize it, see [https://docs.github.com/en/get-started/start-your-journey/setting-up-your-profile](https://docs.github.com/en/get-started/start-your-journey/setting-up-your-profile) 
3. And after you can create your own project by the technology you want see here to see more about it [https://docs.github.com/en/get-started/start-your-journey/hello-world](https://docs.github.com/en/get-started/start-your-journey/hello-world)
4. You can see popular project too and clone them and see how they works, see that here [https://github.com/trending](https://github.com/trending)

Have a good start and enjoy github !

**Reply by SilasChalwe** [reply]
Thank you 👍

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @SilasChalwe ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by SilasChalwe** [reply]
Ooh, alright thanks 🙏

**Comment by SilasChalwe** [comment]
Thank you On Aug 5, 2024 15:37, Lili ***@***.***> wrote:
Hey there!  👋
Thanks for posting in the GitHub Community, @SilasChalwe ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
I've gone ahead and moved this to the correct category for you. Good luck!
